src/client/entity-editor/common/language-field.tsx

```diff
@@ -56,14 +57,16 @@ function LanguageField({
 	const label =
 		<ValidationLabel empty={empty} error={error}>Language</ValidationLabel>
 	;
-
+	const MAX_F2_OPTIONS = 20;
```

Comment:
Why limit this to 20?

Reply:
As discussed showing top 20ish (frequency >=2) languages in language field should be enough

Reply:
I think there are currently only 21 languages with frequency = 2  in the DB, so I thought maybe it was related.
I'd say let's not limit the number of f >= 2 languages and only limit f < 2